src/django_rich/test.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+import sys
+import unittest
+from unittest.result import (  # type: ignore [attr-defined]
+    STDERR_LINE,
+    STDOUT_LINE,
+    failfast,
+)
+
+from django.test import testcases
+from django.test.runner import DebugSQLTextTestResult, DiscoverRunner
+from rich.color import Color
+from rich.console import Console
+from rich.rule import Rule
+from rich.style import Style
+from rich.traceback import Traceback
+
+# PDBDebugResult added in Django 3.0.
+try:
+    from django.test.runner import PDBDebugResult
+except ImportError:
+    PDBDebugResult = None
```

Comment:
Prefer version checking over feature checking: https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

Also better for Mypy to avoid this `try: import ` pattern: https://adamj.eu/tech/2021/12/29/python-type-hints-optional-imports/